gcp-common/src/main/java/io/micronaut/gcp/credentials/GoogleCredentialsConfiguration.java

```diff
@@ -138,4 +149,25 @@ public boolean isEnabled() {
     public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
+
+    /**
+     * If the HttpClient based transport should be used for retrieving authentication
+     * tokens. Default value is {@value #DEFAULT_USE_HTTP_CLIENT}. Note that if HttpClient is not
+     * on the classpath, the GCP SDK's default transport will be used instead.
+     *
+     * @return Whether the {@link HttpClient} should be used for retrieving authentication tokens.
+     */
+    public boolean isUseHttpClient() {
```

Comment:
add `@since` to new methods.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-gcp/pull/1015/commits/d1cf3cb84f0de41f77edb62311c44bfa6aa91c37